### PR TITLE
Support Spring Native configuration for samples

### PR DESCRIPTION
**Pull request by moarychan**
This PR is based on the PR https://github.com/Azure-Samples/azure-spring-boot-samples/pull/209.

## Relate Issue
- https://github.com/Azure/azure-sdk-for-java/issues/28053

## Changelog
- Add a new sample for `spring-cloud-azure-starter-appconfiguration` usage.
- Add a new sample for `spring-cloud-azure-starter-storage-queue` usage.
- Add a new sample for `spring-cloud-azure-starter-eventhubs` usage.
- Add profile **native** and **buildpack** to build spring native image.
- ~Add profile **shaded** to build spring native image when found command too long error, and you should add the configuration `spring.cloud.azure.compatibility-verifier.enabled=false` to disable the version compatibility verifier.~ No need for shade plugin anymore since the new version uses the temp file to transform the commands.
- Add a new sample `storage-blob` to introduce the support for Spring Native.

### CODE REVIEW

File: appconfiguration/spring-cloud-azure-starter-appconfiguration/appconfiguration-sample/README.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+```shell
+terraform\setup_env.ps1
+```
+
+#### Run with Command Prompt or Native Tools Command Prompt
+
+```shell
+terraform\setup_env.bat
+```
```
Comment (saragluna) [inline appconfiguration/spring-cloud-azure-starter-appconfiguration/appconfiguration-sample/README.md:124]: should we only stick to bash and powershell?

File: appconfiguration/spring-cloud-azure-starter-appconfiguration/appconfiguration-sample/README.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+The terraform destroy command terminates resources managed by your Terraform project.   
+To destroy the resources you created.
+
+#### Run with Bash
+
+```shell
+terraform -chdir=./terraform destroy -auto-approve
+```
+
+#### Run with Powershell, Command Prompt or Native Tools Command Prompt
```
Comment (saragluna) [inline appconfiguration/spring-cloud-azure-starter-appconfiguration/appconfiguration-sample/README.md:222]: We should keep the same style as other file

File: cosmos/spring-cloud-azure-starter-cosmos/spring-cloud-azure-cosmos-sample/pom.xml [hunk 3]
```diff
@@ -17,6 +17,11 @@
   <name>Read and Write Data with Azure Cosmos DB Using CosmosClient</name>
   <description>Sample project for Spring Cloud Azure Starter Cosmos</description>
 
+  <properties>
+    <imageName>spring-cloud-azure-cosmos-sample</imageName>
+    <mainClass>com.azure.spring.sample.cosmos.CosmosSampleApplication</mainClass>
+  </properties>
+
```
Comment (saragluna) [inline cosmos/spring-cloud-azure-starter-cosmos/spring-cloud-azure-cosmos-sample/pom.xml:24]: why do we need this?

File: eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/README.md [hunk 4]
```diff
@@ -0,0 +1,317 @@
+# Using Spring Cloud Stream Binder for Azure Event Hubs
+
+This code sample demonstrates how to use the `Spring Cloud Stream Binder` for `Azure Event Hubs`.The
+sample app has two operating modes.
+One way is to expose a Restful API to receive string message, another way is to automatically provide string messages.
+These messages are published to one `Event Hub` instance and then consumed by one consumer
```
Comment (saragluna) [inline eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/README.md:6]: `Event Hubs`

File: eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/README.md [hunk 5]
```diff
@@ -0,0 +1,317 @@
...
+# In the root directory of the sample
+# Initialize your Terraform configuration
+terraform -chdir=./terraform init
+
+# Apply your Terraform Configuration
+terraform -chdir=./terraform apply -auto-approve
+
+```
+
+#### Run with Powershell, Command Prompt or Native Tools Command Prompt
```
Comment (saragluna) [inline eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/README.md:81]: same here

File: eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/README.md [hunk 6]
```diff
@@ -0,0 +1,317 @@
...
+
+Users can get all the supported EventHubs message headers [here](https://microsoft.github.io/spring-cloud-azure/4.0.0-beta.4/4.0.0-beta.4/reference/html/index.html#scs-eh-headers) to configure.
+
+### Resource Provision
+
+Event Hubs binder supports provisioning of event hub and consumer group, users could use [properties](https://microsoft.github.io/spring-cloud-azure/4.0.0-beta.4/4.0.0-beta.4/reference/html/index.html#resource-provision) to enable provisioning.
+
+### Partitioning Support
+
+A PartitionSupplier with user-provided partition information will be created to configure the partition information about the message to be sent. The binder supports Event Hubs partitioning by allowing setting partition key and id. Please refer to the [reference doc](https://microsoft.github.io/spring-cloud-azure/4.0.0-beta.4/4.0.0-beta.4/reference/html/index.html#partitioning-support-2) for more details.
```
Comment (saragluna) [inline eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/README.md:313]: wrong link

File: eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/src/main/java/com/azure/spring/sample/eventhubs/client/EventHubClientApplication.java [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * @author Warren Zhu
+ */
```
Comment (saragluna) [inline eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/src/main/java/com/azure/spring/sample/eventhubs/client/EventHubClientApplication.java:25]: remove this

**Comment by moarychan** [comment]
https://github.com/Azure-Samples/azure-spring-boot-samples/issues/291

File: eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md [hunk 8]
```diff
@@ -74,7 +74,7 @@ terraform -chdir=./terraform apply -auto-approve
 
 ```
 
-#### Run with Powershell
+#### Run with Powershell, Command Prompt or Native Tools Command Prompt
```
Comment (saragluna) [inline eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md:77]: same here

File: eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md [hunk 9]
```diff
@@ -133,7 +133,13 @@ source ./terraform/setup_env.sh
...
 
 ```shell
-. terraform\setup_env.ps1
+terraform\setup_env.ps1
+```
+
+#### Run with Command Prompt or Native Tools Command Prompt
+
+```shell
+terraform\setup_env.bat
```
Comment (saragluna) [inline eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md:142]: do we need this

**Review by saragluna — CHANGES_REQUESTED** [review]

File: appconfiguration/spring-cloud-azure-starter-appconfiguration/appconfiguration-sample/README.md (hunk 1 above)
Comment (moarychan) [reply appconfiguration/spring-cloud-azure-starter-appconfiguration/appconfiguration-sample/README.md:124]: The commands of terraform for Windows Command Prompt is the same with Powershell except for the env setup script, I think lots of users will use this usage.

File: appconfiguration/spring-cloud-azure-starter-appconfiguration/appconfiguration-sample/README.md (hunk 2 above)
Comment (moarychan) [reply appconfiguration/spring-cloud-azure-starter-appconfiguration/appconfiguration-sample/README.md:222]: I discuss with @backwind1233 , it is the same with Powershell commands, so use one title to reduce the duplicate code.

File: cosmos/spring-cloud-azure-starter-cosmos/spring-cloud-azure-cosmos-sample/pom.xml (hunk 3 above)
Comment (moarychan) [reply cosmos/spring-cloud-azure-starter-cosmos/spring-cloud-azure-cosmos-sample/pom.xml:24]: I got the `command too long` exception when using native-image command, the solution is to use the shade profile. These two properties are used for the shade maven plugin to define the main class, if the user uses the native profile is OK then can ignore the shade profile and the properties.

File: eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md (hunk 9 above)
Comment (moarychan) [reply eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md:142]: It's required to set the env in `x64 Native Tools Command Prompt`, it's an enhancement tool for Windows Command Prompt.

**Review by backwind1233 — DISMISSED** [review]
LGTM

File: eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-binder/pom.xml [hunk 10]
```diff
@@ -3,45 +3,41 @@
...
+      <artifactId>spring-cloud-azure-stream-binder-eventhubs</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-logging</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.hibernate.validator</groupId>
+      <artifactId>hibernate-validator</artifactId>
+    </dependency>
```
Comment (saragluna) [inline eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-binder/pom.xml:37]: why do we need this?

File: eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-multibinders/pom.xml [hunk 11]
```diff
@@ -3,39 +3,37 @@
...
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-logging</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.hibernate.validator</groupId>
+      <artifactId>hibernate-validator</artifactId>
+    </dependency>
+  </dependencies>
```
Comment (saragluna) [inline eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-multibinders/pom.xml:38]: same here

File: pom.xml [hunk 12]
```diff
@@ -27,6 +27,14 @@
...
     <version.spring-cloud-feature-management>2.4.0</version.spring-cloud-feature-management>
     <version.azure-spring-boot-starter-keyvault-certificates>3.14.0</version.azure-spring-boot-starter-keyvault-certificates>
+    <version.spring-cloud-azure-native-configuration>4.0.0-beta.1</version.spring-cloud-azure-native-configuration>
+    <maven.surefire.plugin.version>3.0.0-M5</maven.surefire.plugin.version>
+    <exec.maven.plugin.version>3.0.0</exec.maven.plugin.version>
+    <version.spring-native>0.11.4</version.spring-native>
+    <native.maven.plugin.version>0.9.11</native.maven.plugin.version>
+    <maven.shade.plugin.version>3.2.4</maven.shade.plugin.version>
+    <repackage.classifier>exec</repackage.classifier>
+    <imageName>${artifactId}</imageName>
```
Comment (saragluna) [inline pom.xml:37]: can we add these properties to the native profile?

File: spring-native/storage-blob/README.md [hunk 13]
```diff
@@ -0,0 +1,255 @@
...
+- System Requirements
+
+GraalVM `native-image` compiler should be installed, see [System Requirements](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#getting-started-native-image-system-requirements) for more details. If using the Windows platform, you need to install `Visual Studio Build Tools`.
+
+- Build the native application
+
+Run `mvn -Pnative -DskipTests package` command using `x64 Native Tools Command Prompt`, see [Build the native application](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#_build_the_native_application_2) for more details.
+
+```shell
+mvn -Pnative -DskipTests package
```
Comment (backwind1233) [inline spring-native/storage-blob/README.md:194]: It seems I can't run it in my local machine.

File: spring-native/storage-blob/README.md [hunk 14]
```diff
@@ -0,0 +1,255 @@
...
+
+## What You Will Build
+You will build an application that use Spring Resource abstraction to read and write data with [Azure Storage Blob](https://azure.microsoft.com/services/storage/blobs/).
+
+## What You Need
+
+- [An Azure subscription](https://azure.microsoft.com/free/)
+- [Terraform](https://www.terraform.io/)
+- [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli)
+- [JDK8](https://www.oracle.com/java/technologies/downloads/) or later
```
Comment (saragluna) [inline spring-native/storage-blob/README.md:13]: Is this correct?

File: spring-native/storage-blob/README.md [hunk 15]
```diff
@@ -0,0 +1,255 @@
...
+mvn clean spring-boot:run
+```
+
+### Run the sample in IDEs
+
+You can debug your sample by adding the saved output values to the tool's environment variables or the sample's `application.yaml` file.
+
+* If your tool is `IDEA`, please refer to [Debug your first Java application](https://www.jetbrains.com/help/idea/debugging-your-first-java-application.html) and [add environment variables](https://www.jetbrains.com/help/objc/add-environment-variables-and-program-arguments.html#add-environment-variables).
+
+* If your tool is `ECLIPSE`, please refer to [Debugging the Eclipse IDE for Java Developers](https://www.eclipse.org/community/eclipse_newsletter/2017/june/article1.php) and [Eclipse Environment Variable Setup](https://examples.javacodegeeks.com/desktop-java/ide/eclipse/eclipse-environment-variable-setup-example/).
```
Comment (saragluna) [inline spring-native/storage-blob/README.md:152]: Maybe we can remove this two here, just to demo how to run as native?

File: spring-native/storage-blob/pom.xml [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+      <artifactId>spring-boot-starter-logging</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.azure.spring</groupId>
+      <artifactId>spring-cloud-azure-starter-storage-blob</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.azure.spring</groupId>
+      <artifactId>spring-cloud-azure-starter-actuator</artifactId>
+    </dependency>
```
Comment (saragluna) [inline spring-native/storage-blob/pom.xml:41]: do we need this?

File: storage/spring-cloud-azure-starter-storage-blob/storage-blob-sample/pom.xml [hunk 17]
```diff
@@ -26,5 +26,13 @@
...
       <artifactId>spring-boot-starter-web</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-logging</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.azure.spring</groupId>
+      <artifactId>spring-cloud-azure-starter-actuator</artifactId>
+    </dependency>
```
Comment (saragluna) [inline storage/spring-cloud-azure-starter-storage-blob/storage-blob-sample/pom.xml:36]: Why do we need the actuator?

File: storage/spring-cloud-azure-starter-storage-queue/storage-queue-sample/src/main/java/com/azure/spring/sample/storage/queue/StorageQueueApplication.java [hunk 18]
```diff
@@ -0,0 +1,40 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/**
+ * @author Moary Chen
+ */
```
Comment (saragluna) [inline storage/spring-cloud-azure-starter-storage-queue/storage-queue-sample/src/main/java/com/azure/spring/sample/storage/queue/StorageQueueApplication.java:18]: remove this

**Review by saragluna — CHANGES_REQUESTED** [review]

File: eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/pom.xml [hunk 19]
```diff
@@ -0,0 +1,43 @@
...
+      <artifactId>spring-cloud-azure-starter-eventhubs</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-logging</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.azure.spring</groupId>
+      <artifactId>spring-cloud-azure-starter-actuator</artifactId>
+    </dependency>
```
[the remark this answers is not in the document]
Comment (moarychan) [reply eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/pom.xml:37]: No, I will remove it first.

File: eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/src/main/resources/application.yaml [hunk 20]
```diff
@@ -0,0 +1,12 @@
...
+    azure:
+      eventhubs:
+        namespace: ${AZURE_EVENTHUBS_NAMESPACE}
+        event-hub-name: ${AZURE_EVENTHUB_NAME}
+        processor:
+          checkpoint-store:
+            container-name: ${AZURE_STORAGE_CONTAINER_NAME}
+            account-name:  ${AZURE_STORAGE_ACCOUNT_NAME}
+        consumer:
+          consumer-group: ${AZURE_EVENTHUB_CONSUMER_GROUP}
```
[the remark this answers is not in the document]
Comment (moarychan) [reply eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/src/main/resources/application.yaml:12]: OK

File: spring-native/storage-blob/README.md (hunk 14 above)
Comment (moarychan) [reply spring-native/storage-blob/README.md:13]: I have added the `System Requirements` for these two building ways in each section, let me add docker and native-image here.

File: spring-native/storage-blob/README.md (hunk 15 above)
Comment (moarychan) [reply spring-native/storage-blob/README.md:152]: OK

File: storage/spring-cloud-azure-starter-storage-blob/storage-blob-sample/pom.xml (hunk 17 above)
Comment (moarychan) [reply storage/spring-cloud-azure-starter-storage-blob/storage-blob-sample/pom.xml:36]: Removed.

File: storage/spring-cloud-azure-starter-storage-queue/storage-queue-sample/src/main/java/com/azure/spring/sample/storage/queue/StorageQueueApplication.java (hunk 18 above)
Comment (moarychan) [reply storage/spring-cloud-azure-starter-storage-queue/storage-queue-sample/src/main/java/com/azure/spring/sample/storage/queue/StorageQueueApplication.java:18]: OK

File: eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/pom.xml [hunk 21]
```diff
@@ -2,38 +2,38 @@
...
+      <artifactId>spring-cloud-azure-starter-integration-eventhubs</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-logging</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.hibernate.validator</groupId>
+      <artifactId>hibernate-validator</artifactId>
+    </dependency>
```
[the remark this answers is not in the document]
Comment (moarychan) [reply eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/pom.xml:36]: There's a lot of validator dependencies, let me delete these in another PR after this PR merged.

File: eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/src/main/resources/application.yaml [hunk 22]
```diff
@@ -0,0 +1,12 @@
+spring:
+  cloud:
+    azure:
+      eventhubs:
+        namespace: ${AZURE_EVENTHUBS_NAMESPACE}
+        event-hub-name: ${AZURE_EVENTHUB_NAME}
+        consumer:
+          consumer-group: ${AZURE_EVENTHUB_CONSUMER_GROUP}
+        processor:
```
Comment (saragluna) [inline eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/src/main/resources/application.yaml:9]: this is not correct, producer, consumer, processor maps to three different clients, so if the code uses only producer and consumer clients, then the configurations under processor are not required.

File: eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-binder/pom.xml [hunk 23]
```diff
@@ -3,45 +3,37 @@
...
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-logging</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-actuator</artifactId>
+    </dependency>
+  </dependencies>
```
Comment (saragluna) [inline eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-binder/pom.xml:38]: do we need the actuator here?

File: pom.xml [hunk 24]
```diff
@@ -27,6 +27,9 @@
     <version.spring-cloud-appconfiguration-starter>2.5.0</version.spring-cloud-appconfiguration-starter>
     <version.spring-cloud-feature-management>2.4.0</version.spring-cloud-feature-management>
     <version.azure-spring-boot-starter-keyvault-certificates>3.14.0</version.azure-spring-boot-starter-keyvault-certificates>
+    <version.spring-cloud-azure-native-configuration>4.0.0-beta.1</version.spring-cloud-azure-native-configuration>
+    <version.spring-native>0.11.4</version.spring-native>
+    <repackage.classifier>exec</repackage.classifier>
```
Comment (saragluna) [inline pom.xml:32]: why can't we put these properties to the native profile too?

**Review by saragluna — CHANGES_REQUESTED** [review]

File: README.md [hunk 25]
```diff
@@ -47,6 +47,7 @@ We use `main` branch as the develop branch while setting the default branch as t
 | App Configuration| [azure-spring-cloud-feature-management:2.2.0]                       | [feature-management-sample](appconfiguration/feature-management-sample)                                                            |
 | App Configuration| [azure-spring-cloud-feature-management:2.2.0]                       | [feature-management-web-sample](appconfiguration/feature-management-web-sample)                                                    |
 | App Configuration| [azure-spring-cloud-appconfiguration-config:2.3.0]                  | [azure-appconfiguration-conversion-sample-complete](appconfiguration/azure-appconfiguration-conversion-sample-complete)            |
+| App Configuration| [spring-cloud-azure-starter-appconfiguration:4.0.0]                 | [appconfiguration-sample](appconfiguration/spring-cloud-azure-starter-appconfiguration/appconfiguration-sample)                    |
```
[the remark this answers is not in the document]
Comment (moarychan) [reply README.md:50]: Yeah

File: README.md [hunk 26]
```diff
@@ -57,6 +58,7 @@ We use `main` branch as the develop branch while setting the default branch as t
 | Event Hubs       | [spring-cloud-azure-starter-integration-eventhubs:4.0.0]     | [eventhubs-integration](eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration)                          |
 | Event Hubs       | [spring-cloud-azure-stream-binder-eventhubs:4.0.0]           | [eventhubs-binder](eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-binder)                                          |
 | Event Hubs       | [spring-cloud-azure-stream-binder-eventhubs:4.0.0]           | [eventhubs-multibinders](eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-multibinders)                              |
+| Event Hubs       | [spring-cloud-azure-starter-eventhubs:4.0.0]                 | [eventhubs-client](eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client)                                                |
```
[the remark this answers is not in the document]
Comment (moarychan) [reply README.md:61]: Maybe need a discussion and a new PR to unify the naming for all samples.

File: pom.xml (hunk 24 above)
Comment (moarychan) [reply pom.xml:32]: I put the spring-native and spring-cloud-azure-native-configuration dependencies in the parent dependency-management. the `repackage.classifier` is used for spring boot maven plugin, it doesn't need to move.

Comment (moarychan) [reply pom.xml:32]: Maybe use the spring-native and spring-cloud-azure-native-configuration with the version in each profile dependencies enough.

File: eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/README.md [hunk 27]
```diff
@@ -1,10 +1,6 @@
 # Using Spring Cloud Stream Binder for Azure Event Hubs
```
Comment (saragluna) [inline eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/README.md:1]: The title should be updated too

File: eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/README.md [hunk 28]
```diff
@@ -1,10 +1,6 @@
...
 
-This code sample demonstrates how to use the `Spring Cloud Stream Binder` for `Azure Event Hubs`.The
-sample app has two operating modes.
-One way is to expose a Restful API to receive string message, another way is to automatically provide string messages.
-These messages are published to one `Event Hubs` instance and then consumed by one consumer
-endpoint from the same application.
+This code sample demonstrates how to use the client of `Azure SDK Event Hubs` to interact with `Azure Event Hubs`.
```
Comment (saragluna) [inline eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/README.md:3]: how to use `Azure Event Hubs SDK`

**Review by saragluna — CHANGES_REQUESTED** [review]

File: .github/workflows/codeql_analysis.yml [hunk 29]
```diff
@@ -47,7 +47,6 @@ jobs:
           echo "eng" >> .git/info/sparse-checkout
           echo "sdk/keyvault" >> .git/info/sparse-checkout
           echo "sdk/boms" >> .git/info/sparse-checkout
-          echo "sdk/aot" >> .git/info/sparse-checkout
```
Comment (saragluna) [inline .github/workflows/codeql_analysis.yml:50]: Should we change the profile too?

**Review by saragluna — CHANGES_REQUESTED** [review]

**Review by saragluna — APPROVED** [review]
LGTM.

File: spring-native/storage-blob/README.md (hunk 13 above)
Comment (saragluna) [reply spring-native/storage-blob/README.md]: verified, I can run it.

**Pull request merged**